STREAMING/GetPlayerSwitchState.md

```diff
@@ -1,5 +1,6 @@
 ---
 ns: STREAMING
+aliases: ["0x470555300D10B2A5"]
```

Comment:
You should only need to add the alias if you actually changed the function name. Aliases are mainly there to preserve compatibility with game modes that maintain older native names.

Reply:
should be good